search/2015-02-28-Preview/swagger/searchindex.json

```diff
@@ -0,0 +1,280 @@
+        },
+        "queryType": {
+          "$ref": "#/definitions/QueryType",
+          "description": "Gets or sets a value that specifies the syntax of the search query. The default is 'simple'. Use 'full' if your query uses the Lucene query syntax."
+        },
+        "scoringParameters": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Gets or sets the list of parameter values to be used in scoring functions (for example, referencePointParameter) using the format name:value. For example, if the scoring profile defines a function with a parameter called 'mylocation' the parameter string would be \"mylocation:-122.2,44.8\"(without the quotes)."
+        },
+        "scoringProfile": {
+          "type": "string",
+          "description": "Gets or sets the name of a scoring profile to evaluate match scores for matching documents in order to sort the results."
+        },
+        "searchFields": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Gets or sets the list of field names to include in the full-text search."
+        },
+        "searchMode": {
+          "$ref": "#/definitions/SearchMode",
+          "description": "Gets or sets a value that specifies whether any or all of the search terms must be matched in order to count the document as a match."
+        },
+        "select": {
+          "type": "array",
+          "items": {
+            "type": "string"
+          },
+          "description": "Gets or sets the list of fields to retrieve. If unspecified, all fields marked as retrievable in the schema are included."
+        },
+        "skip": {
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the number of search results to skip. This value cannot be greater than 100,000. If you need to scan documents in sequence, but cannot use Skip due to this limitation, consider using OrderBy on a totally-ordered key and Filter with a range query instead."
+        },
+        "top": {
+          "externalDocs": {
+            "url": "https://msdn.microsoft.com/en-us/library/azure/dn798927.aspx"
+          },
+          "type": "integer",
+          "format": "int32",
+          "description": "Gets or sets the number of search results to retrieve. This can be used in conjunction with Skip to implement client-side paging of search results. If results are truncated due to server-side paging, the response will include a continuation token that can be passed to ContinueSearch to retrieve the next page of results. See DocumentSearchResponse.ContinuationToken for more information."
```

Comment:
> See DocumentSearchResponse.ContinuationToken for more information

Since the parent object is marked as x-ms-external: true, I assume DocumentSearchResponse.ContinuationToken in some other document as I couldn't  find it here